<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="192.168.56.102:5555"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="84.923">
  <properties />
            <testcase name="personInfo" time="84.923" classname="cn.crazy.appium.testcases.XinChangTaiTest">
                <failure type="java.lang.AssertionError"
                  message="null"
              >
<![CDATA[java.lang.AssertionError: null
  org.testng.Assert.fail(Assert.java:94)
  org.testng.Assert.fail(Assert.java:101)
  cn.crazy.appium.testng.Assertion.fail(Assertion.java:63)
  cn.crazy.appium.testng.Assertion.assertEquals(Assertion.java:19)
  cn.crazy.appium.page.PersonInofPage.personChange(PersonInofPage.java:130)
  cn.crazy.appium.testcases.XinChangTaiTest.personInfo(XinChangTaiTest.java:31)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:483)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:882)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1189)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
  org.testng.SuiteRunner.access$000(SuiteRunner.java:38)
  org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:382)
  org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
  java.util.concurrent.FutureTask.run(FutureTask.java:266)
  java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
  java.lang.Thread.run(Thread.java:745)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
